applications/sdp/gpio/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _sdp_gpio:
+
+SDP GPIO
+########
+
+This application demonstrates how to use SDP on FLPR in Zephyr threadless mode by implementing a subset of Zephyr GPIO API.
```

Comment:
I think "creating/implementing SDP" would be a better wording than "use SDP". I wouldn't say that FLPR application is a good place to check how to use a SDP, because they would be used similarly to regular peripherals/drivers. In the worst case someone could read the APP side code for this purpose, but not the FLPR side. Unless I'm getting something wrong.